classes/paladin.lua

```diff
@@ -4,6 +4,11 @@ local function registerClass(self)
     local flashOfLight = GetSpellInfo(19750);
     local exorcism = GetSpellInfo(879);
     local holyLight = GetSpellInfo(635);
+	local how = GetSpellInfo(24275);
+
+	-- Hammer of Light
+	self:RegisterAura("how", 0, how, nil, "", 0, 0, 0, 0, false, { how });
+	self:RegisterCounter("how");

```

Comment:
Please be careful with leading tabulations. These should be whitespaces instead.

Reply:
Fixed